platform/understand/externally-vs-platform-deployed.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Virtual Cluster Lifecycle Ownership
+sidebar_label: Virtual Cluster Lifecycle Ownership
+---
+
+# Virtual Cluster Lifecycle Ownership
+
+There are two main types of virtual clusters based on how they're managed: platform-managed and externally managed. These classifications determine how the clusters are created, updated, and integrated with the platform's features. Understanding the differences between these types is important for effective cluster management and utilization of platform capabilities.
+
+## Platform-Managed Virtual Clusters
+
+These are virtual clusters that are deployed and fully managed through our platform. They benefit from the platform's complete set of management features and automated lifecycle management.
+
+:::info Characteristics
+- Identified by `external: false` in `VirtualClusterInstance` CR
```

Comment:
I wouldn't expose this as it can cause unintended changes if people try and manipulate this themselves.